app/[lang]/chapters/[slug]/[lesson]/layout.tsx

```diff
@@ -1,34 +1,29 @@
 'use client'
 
-import { useLessonUnlocked } from 'hooks'
+import { useLessonUnlocked, useLocalStorage } from 'hooks'
 import { useRouter } from 'next/navigation'
-import { useEffect, useState } from 'react'
 import Navbar from 'components/Navbar'
 import { lessons } from 'content'
 
 export default function Layout({ children, params }) {
   const router = useRouter()
-  const result = useLessonUnlocked()
-  const [displayLesson, setDisplayLesson] = useState<boolean>(false)
+  const userInfo = useLocalStorage('SavingSatoshiUser')
```

Comment:
Do we really need a hook for this? Whats the advantage of this over calling `LocalStorage.getItem('SavingSatoshiUser')` directly here?

This would be a nice addition if we could pass a fallback value if the value for the given key couldn't be found but otherwise I don't see the point of introducing a hook.